pronto_mia/lib/ui/views/user/edit/user_edit_viewmodel.dart

```diff
@@ -125,13 +110,43 @@ class UserEditViewModel extends FormViewModel {
       case 'canEditUsers':
         accessControlList.canEditUsers = value;
         if (value) {
+          accessControlList.canViewUsers = true;
+          accessControlList.canViewDepartments = true;
+          accessControlList.canEditDepartments = true;

```

Comment:
Why is this linked to `CanEditUsers`? It is not necessary in order to change a users department.